feat: update docs (#9)
time-loop/octoherd-script-renovate-library-major-version-driver: README.md

node cli.js \
  -R time-loop/*-cdk \
  -T ghp_0123456789abcdefghijABCDEFGHIJabcdefgh \
  --majorVersion v11
```

## Options

| option            | type   | default | description        |
| ----------------- | ------ | ------- | ------------------ |
| `--majorVersion`  | string | none    | Major version number for the library, for example `v11`. If you provide `all` then it will instead address the `all non-major updates` PR. If you provide `projen`, it will address the `fix(deps): upgrade projen` PR. |
| `--library`       | string | `@time-loop/cdk-library` | Full name of library to be updated via renovate |

> | `--maxAgeDays`    | number | 7 | The maximum age, in days, since when a PR was merge to consider it the relevant PR. Ignored except when doing `all non-major updates` |

Reviewer: ```suggestion
| `--maxAgeDays`    | number | 7 | The maximum age, in days, since when a PR was merge to consider it the relevant PR. Only used by the special cases of `majorVersion` |
```
